api/base/filters.py

```diff
@@ -27,6 +27,11 @@ class ODMOrderingFilter(OrderingFilter):
     def filter_queryset(self, request, queryset, view):
         ordering = self.get_ordering(request, queryset, view)
         if ordering:
+            if isinstance(queryset, list) and isinstance(ordering, tuple):
```

Comment:
We might want to do `isinstance(x, (list, tuple)` for both the queryset and `ordering`.